include/OpenSpaceToolkit/Astrodynamics/Flight/System/Dynamics/Thruster/ConstantThrustThruster.hpp

```diff
@@ -0,0 +1,167 @@
+
+using ostk::math::obj::Vector3d;
+using ostk::math::geom::d3::trf::rot::RotationMatrix;
+
+using ostk::physics::time::Instant;
+using ostk::physics::units::Mass;
+
+using ostk::astro::flight::system::SatelliteSystem;
+using ostk::astro::flight::system::PropulsionSystem;
+using ostk::astro::flight::system::Dynamics;
+using ostk::astro::trajectory::State;
+using ostk::astro::trajectory::LocalOrbitalFrameDirection;
+
+/// @brief                      Define the acceleration experienced by a point mass due to a constant thrust thruster
+/// dynamics
+
+class ConstantThrustThruster : public Thruster
+{
+   public:
+    /// @brief              Constructor
+    ///
+    /// @code
+    ///                     const String aName = { ... };
+    ///                     const SatelliteSystem aSatelliteSystem = { ... };
+    ///                     const Direction aThrustDirection = { ... };
+    ///                     ConstantThrustThruster constantThrustThrusterDynamics = { aName, aSatelliteSystem,
+    ///                     aThrustDirection};
+    /// @endcode
+    ///
+    /// @param              [in] aDirection A direction
+
+    ConstantThrustThruster(
+        const SatelliteSystem& aSatelliteSystem,
+        const LocalOrbitalFrameDirection& aThrustDirection,
+        const String& aName = String::Empty()
+    );
+
+    /// @brief              Destructor
+
+    virtual ~ConstantThrustThruster() override;
+
+    /// @brief              Clone constant thrust thruster dynamics
+    ///
+    /// @return             Pointer to cloned constant thrust thruster dynamics
+
+    virtual ConstantThrustThruster* clone() const override;
+
+    /// @brief              Equal to operator
+    ///
+    /// @param              [in] aConstantThrustThruster A constant thrust thruster dynamics
+    /// @return             True if constant thrust thruster dynamics are equal
+
+    bool operator==(const ConstantThrustThruster& aConstantThrustThruster) const;
+
+    /// @brief              Not equal to operator
+    ///
+    /// @param              [in] aThruster A constant thrust thruster dynamics
+    /// @return             True if constant thrust thruster dynamics are not equal
+
+    bool operator!=(const ConstantThrustThruster& aConstantThrustThruster) const;
+
+    /// @brief              Output stream operator
+    ///
+    /// @param              [in] anOutputStream An output stream
+    /// @param              [in] aThruster A constant thrust thruster dynamics
+    /// @return             A reference to output stream
+
+    friend std::ostream& operator<<(
+        std::ostream& anOutputStream, const ConstantThrustThruster& aConstantThrustThruster
+    );
+
+    /// @brief              Check if constant thrust thruster dynamics is defined
+    ///
+    /// @return             True if constant thrust thruster dynamics is defined
+
+    virtual bool isDefined() const override;
+
+    /// @brief              Get constant thrust thruster dynamics' thrust
```

Comment:
```suggestion
    /// @brief              Get thrust 
```
Nit: The brief is a bit cyclical.